docs/web/sdk-component-interactions.mdx

```diff
@@ -6,142 +6,67 @@ description: Geocortex Web - Learn how to make custom components interact in Geo
-
-## Prerequisites
-
-Check out [implementing a custom component](usecases-implement-component-with-ui.mdx) to learn the basics of implementing components.
-
-## Overview
-
 <!-- TODO-API direct link to map model -->
 
-Components have multiple ways of interacting with other components and services. When possible, interaction through [commands and operations](usecases-components-commands-operations.mdx#running-commands-and-operations) should be preferred, as this introduces the loosest coupling. When you need stronger coupling and direct access to the API, then model imports or service injection can be used. A good example of this is needing direct access to the [ArcGIS Map](https://developers.arcgis.com/javascript/latest/api-reference/esri-Map.html), which can be exposed by importing the map model.
+Components have multiple ways of interacting with other components and services. When possible, interaction through [commands and operations](sdk-components-commands-operations.mdx#running-commands-and-operations) should be preferred, as this introduces the loosest coupling. When you need stronger coupling and direct access to the API, then **model imports** or **service injection** can be used. A good example of this is needing direct access to the [ArcGIS Map](https://developers.arcgis.com/javascript/latest/api-reference/esri-Map.html), which can be exposed by importing the map model.
 
-## Interacting with other Components through Models
+## Interacting with Components through Models
 
 <!-- TODO-API link to map model and scalebar/compass components once available -->
 
 Components can interact with each other through their [respective models](concepts-layout.mdx#models). A model can either declare that it is exported, like the map model, or that it imports a model. Components like the `<scalebar>` or `<compass>` import the map model, and this is what allows them to respond to changes in the maps state.
 
-### Example: Component that Adds a Graphics Layer to the Map
-
-The following component has a model which imports the map model as a dependency, and then uses that model to add a graphics layer to the [ArcGIS Map object](https://developers.arcgis.com/javascript/latest/api-reference/esri-Map.html).
-
-:::note
-The property decorated by `@importModel` is completely managed by Geocortex Web and should never be modified or serialized in app config. Its value can change at any time, e.g. to and from undefined, or from one model instance to a completely different one. The component needs to react appropriately to these changes and not make any assumptions about its current value.
-:::
-
-<Tabs
-  defaultValue="component"
-  values={[
-    { label: 'Component', value: 'component', },
-    { label: 'Model', value: 'model', },
-    { label: 'Layout', value: 'layout', },
-    { label: 'UI', value: 'ui', },
-  ]
-}>
-<TabItem value="component">
+```ts
+@exportModel
```

Comment:
I just realized i don't actually know what this does. Above we briefly mention it, but don't explain when or why we would use this decorator. I tried following the `respective models` link but it didn't contain the info.

Reply:
Added a better description of how import and export relate, and better links to the appropiate component models and  model binding in the layout articles